index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Technical Documentation</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <nav id="navbar">
+    <header>Something about Sofware development</header>
+    <ul>
+      <li><a class="nav-link" href="#HTML">Html</a></li>
+      <li>
+        <a class="nav-link" href="#CSS">CSS</a>
+      </li>
+      <li>
+        <a class="nav-link" href="#JavaScript">JavaScript</a>
+      </li>
+      <li><a class="nav-link" href="#Scratch">Scratch</a></li>
+      <li><a class="nav-link" href="#Python">Python</a></li>
+      <li>
+        <a class="nav-link" href="#C++">C++</a>
+      </li>
+    </ul>
+  </nav>
+
+  <main id="main-doc">
+    <!-- First Section-->
+    <section class="main-section" id="HTML">
+      <header>HTML</header>
```

Comment:
This part is fine. I recommend that you use the```<h2>``` tag in the section header. You could implement it in this way. 
```<header><h2>HTML</h2></header>```